### ISSUE TITLE
HI, CAN I CHANGE THE LANGUAGE?

### ISSUE THREAD

**Issue by DeterminantMatrix** [razem-io/OCRmyPDFonWEB#1]
DIDN'T FIND THE BUTTON, THANKS

**Comment by Magnusretrotech** [comment]
It currently does not support any languages except English. You can translate it manually and submit a pull request if you want support for your language.

**Comment by razem-io** [comment]
No, currently there is no support for different languages. I am open to PRs.
